ruma-federation-api/src/event/get_event/v1.rs

```diff
@@ -0,0 +1,53 @@
+//! [GET /_matrix/federation/v1/event/{eventId}](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-federation-v1-event-eventid)
+
+use ruma_api::ruma_api;
+use ruma_events::pdu::Pdu;
+use ruma_identifiers::{EventId, ServerNameBox};
+use std::time::SystemTime;
+
+ruma_api! {
+    metadata: {
+        description: "Retrieves a single event.",
+        method: GET,
+        name: "get_event",
+        path: "/_matrix/federation/v1/event/:event_id",
+        rate_limited: false,
+        requires_authentication: true,
+    }
+
+    #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+    request: {
+        /// The event ID to get.
+        #[ruma_api(path)]
+        pub event_id: &'a EventId,
+    }
+
+    #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+    response: {
+        /// The `server_name` of the homeserver sending this transaction.
+        pub origin: ServerNameBox,
+
+        /// POSIX timestamp in milliseconds on originating homeserver when this
+        /// transaction started.
```

Comment:
On-the-wire format doesn't matter for Ruma users :)

```suggestion
        /// Time on originating homeserver when this transaction started.
```